### PR TITLE
Milestone: Withdrawal

### PR DESCRIPTION
**Pull request by ECWireless**
- [x]  Before wallet is connected state
- [x]  After wallet is connected state
- [x]  Farm selected state
- [x]  Enable withdrawal when amount is entered
- [x] Withdrawal functionality
- [x]  Confirm deposit modal
- [x]  Add transaction modals

### CODE REVIEW

File: src/components/Modals/ConfirmTransaction/components/index.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+  button {
+    background: transparent;
+    border: none;
+
+    ${(props) =>
+      props.copied &&
+      css`
+        background: #000;
+        border-radius: 5px;
```
Comment (Evalir) [inline src/components/Modals/ConfirmTransaction/components/index.ts]: We can make this `${GU / 2}` to stick with the scale. 😄

File: src/components/Modals/ConfirmTransaction/index.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+import { colors, GU } from 'components/theme';
+
+import { ReactComponent as CoinSvg } from 'assets/icons/coin.svg';
+import { ReactComponent as CloseSvg } from 'assets/icons/close.svg';
+import { ReactComponent as CopySvg } from 'assets/icons/copy.svg';
+import { ReactComponent as DepositButtonActiveSvg } from 'assets/icons/deposit_button_active.svg';
+import { ReactComponent as FarmSvg } from 'assets/icons/farm.svg';
+import { ReactComponent as MultiplierSvg } from 'assets/icons/multiplier.svg';
+import { ReactComponent as RewardsGraphSvg } from 'assets/icons/rewards_graph.svg';
+import { ReactComponent as WithdrawButtonActiveSvg } from 'assets/icons/withdraw_button_active.svg';
```
Comment (Evalir) [inline src/components/Modals/ConfirmTransaction/index.tsx:12]: Did we run these SVGs through an optimizer tool? If not, we should!

File: src/views/DepositWithdraw/components/EnterAmount/index.tsx [hunk 3]
```diff
@@ -21,37 +23,38 @@ import {
...
       setIsDisabled(true);
+      setFarmSelected(false);
     } else {
       setIsDisabled(false);
       setFarmSelected(true);
     }
-  }, [address, setFarmSelected, wpoktInputValue]);
+  }, [address, setFarmSelected, inputValue]);
 
   // This is a placeholder, which will eventually launch the transaction status modal
```
Comment (Evalir) [inline src/views/DepositWithdraw/components/EnterAmount/index.tsx]: We can probably remove this now :)

File: src/views/DepositWithdraw/components/WithdrawInfo/index.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          css={`
+            align-items: center;
+            background: #000;
+            display: flex;
+            padding: ${7 * GU}px;
+          `}
+        >
+          <div
+            css={`
+              box-sizing: border-box;
```
Comment (Evalir) [inline src/views/DepositWithdraw/components/WithdrawInfo/index.tsx]: Hmm, this should actually be a property set at the root level of the page's `css`... Have we been using `content-box` this entire time? 😅

**Review by Evalir — APPROVED** [review]
Haven't tested manually yet, but this is looking good!

File: src/components/Modals/ConfirmTransaction/components/index.ts (hunk 1 above)
Comment (ECWireless) [reply src/components/Modals/ConfirmTransaction/components/index.ts]: Fixed in 71975a4375568e3aabea19bccf7b2865a5342264

File: src/views/DepositWithdraw/components/EnterAmount/index.tsx (hunk 3 above)
Comment (ECWireless) [reply src/views/DepositWithdraw/components/EnterAmount/index.tsx]: 😅 71975a4375568e3aabea19bccf7b2865a5342264

File: src/views/DepositWithdraw/components/WithdrawInfo/index.tsx (hunk 4 above)
Comment (ECWireless) [reply src/views/DepositWithdraw/components/WithdrawInfo/index.tsx]: Oh, yeah that actually is in fact set [here](https://github.com/pokt-network/wpokt-app/blob/milestone/withdrawal/src/index.css#L4), so I'll remove the useless line. Fixed in 71975a4375568e3aabea19bccf7b2865a5342264

File: src/components/Modals/ConfirmTransaction/index.tsx (hunk 2 above)
Comment (ECWireless) [reply src/components/Modals/ConfirmTransaction/index.tsx:12]: Consider them optimized! Fixed in 384a31f3236bfb0c116da635e532d0e2f0383fa1

**Pull request merged**